README.md

```diff
@@ -1,27 +1,28 @@
+[![Gem Version](https://badge.fury.io/rb/rmagick4j.svg)](https://badge.fury.io/rb/rmagick4j)
+[![Build Status](https://travis-ci.org/Serabe/RMagick4J.svg?branch=master)](https://travis-ci.org/Serabe/RMagick4J)
+
 RMagick4J
 =========
 
-RMagick is a Ruby binding to ImageMagick and GraphicsMagick.
-RMagick4J implements ImageMagick functionality and the C portions of RMagick for
-use with JRuby. 
+[RMagick](https://github.com/rmagick/rmagick) is a Ruby binding to ImageMagick
+and GraphicsMagick.  **RMagick4J** implements ImageMagick functionality and the
+C portions of RMagick for use with JRuby.
 
 ## Authors
 
-This project was written by Thomas Palmer, Sergio Rodríguez Arbeo, and Thomas
-Enebo with lots of help from the JRuby community.
+This project was written by Thomas Palmer, Sergio Rodríguez Arbeo, Thomas
```

Comment:
Can you please remove the `Rodríguez` here? It is no longer there in my name. Thank you so much.